lib/livebook/intellisense.ex

```diff
@@ -207,7 +221,96 @@ defmodule Livebook.Intellisense do
       insert_text: name
     }
 
-  @ordered_kinds [:field, :variable, :module, :struct, :interface, :function, :type]
+  defp keyword_macro?(name) do
+    def? = name |> Atom.to_string() |> String.starts_with?("def")
+
+    def? or
+      name in [
+        :alias,
+        :import,
+        :require,
+        :use,
+        :case,
+        :cond,
+        :for,
+        :if,
+        :quote,
+        :raise,
+        :receive,
+        :throw,
+        :try,
+        :unless,
+        :with
+      ]
```

Comment:
For completeness, this is Elixir's list: https://github.com/elixir-lang/elixir/blob/main/lib/elixir/lib/code/formatter.ex#L57-L134

Reply:
Should we paste all of those? Other than `quote` probably makes sense.

Reply:
Nevermind, I meant `unquote`.

I synced with special forms and kernel. I think we don't need the test-related names since those will always be used in a test module and we wouldn't complete those.